README.md

```diff
@@ -1,20 +1,25 @@
 # hls-monitor
 
-Service to monitor one or more hls streams for manifest errors and inconsistencies.  
+Service to monitor one or more hls streams for manifest errors and inconsistencies.
+Possible inconsistencies and errors are:
+  - Media sequence counter issues. 
+  - Discontinuity sequence counter issues. 
+  - Media sequence length should be consistent. The default is 6000ms but can be configured via the env `HLS_MONITOR_INTERVAL`.
```

Comment:
This means that it should be updated at least every 6000 ms and if nothing has happened for 6000 ms it is considered a "stale" manifest?

Reply:
Yes that is correct.

Reply:
Maybe write something like. Detect stale manifests or something like that. But we can fix that later. This is good for now